bokehjs/src/lib/embed/notebook.ts

```diff
@@ -57,6 +57,27 @@ function _init_comms(target: string, doc: Document): void {
     } catch (e) {
       logger.warn(`Jupyter comms failed to register. push_notebook() will not function. (exception reported: ${e})`)
     }
+  } else if  (typeof google != 'undefined' && google.colab.kernel != null) {
+    logger.info(`Registering Google Colab comms for target ${target}`)
+    const comm_manager = google.colab.kernel.comms
+    try {
+      comm_manager.register_target(target, async (comm: Comm) => {
+        logger.info(`Registering Google Colab comms for target ${target}`)
+        const r = new Receiver()
+        for await (const message of comm.messages) {
```

Comment:
@mattpap The `for await ... of` syntax should be fine right? 

```suggestion
        for await (const message of comm.messages) {
```

Reply:
Perfectly fine. Though IE will need a polyfill for `Symbol.asyncIterator`.

Reply:
@mattbest Is IE still supported? In which case where do I put the polyfill?